scheduler/pkg/agent/rproxy_grpc_test.go

```diff
@@ -316,12 +299,13 @@ func TestAddRequestIdToTrailer(t *testing.T) {
 			rp := reverseGRPCProxy{
 				logger: log.New(),
 			}
-			rp.addRequestIdToTrailer(test.ctx, test.trailer)
-			data := test.trailer.Get(util.RequestIdHeader)
```

Comment:
As we are removing the trailer tests? could we add them somewhere else?

Reply:
The function had been removed